src/hypothesis/extra/datetime.py

```diff
@@ -276,6 +276,30 @@ def datetimes(allow_naive=None, timezones=None, min_year=None, max_year=None):
     )
 
 
+def dates(min_year=None, max_year=None):
```

Comment:
hypothesis.strategies has a 'defines_strategy' decorator that should be used here that fixes up the repr of the returned strategy to correspond to the function that generates it. This should be used here and on times.